Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Build Stage
+FROM rustlang/rust:nightly-buster-slim as builder
+
+WORKDIR /app
+RUN set -ex; \ 
+  apt-get update; \
+  apt-get install -y --no-install-recommends \
+  pkg-config \
+  ca-certificates libssl-dev \
+  && rm -rf /var/lib/apt/lists/*
+
+COPY . .
+RUN cargo build --release
+
+# Image Stage
+FROM debian:buster-slim
+
+RUN useradd rust
+
+WORKDIR "/gateway"
+
+ENV ROCKET_ENV=production
```

Comment:
Maybe everything on the same line? `ROCKET_ENV=production ROCKET_ADDRESS=0.0.0.0 ROCKET_PORT=3666`